src/test/java/cholog/wiseshop/domain/campaign/CampaignServiceTest.java

```diff
@@ -124,4 +125,58 @@ public void cleanUp() {
         assertThatThrownBy(() -> campaignService.readCampaign(wrongId))
                 .isInstanceOf(IllegalArgumentException.class);
     }
+    
+    @Test
+    void 캠페인_시작_상태_변경_성공() throws InterruptedException {
+        //given
+        String name = "보약";
+        String description = "먹으면 기분이 좋아져요.";
+        int price = 10000;
+        CreateProductRequest productRequest = new CreateProductRequest(name, description, price);
+        Long productId = productService.createProduct(productRequest);
+
+        //when
+        LocalDateTime startDate = LocalDateTime.now().plusSeconds(1);
+        LocalDateTime endDate = LocalDateTime.now().plusSeconds(10);
+        int goalQuantity = 5;
+
+        Long campaignId = campaignService.createCampaign(
+                new CreateCampaignRequest(startDate, endDate, goalQuantity, productId));
+        Campaign findCampaign = campaignRepository.findById(campaignId).orElseThrow();
+
+        // then
+        Thread.sleep(2000);

```

Comment:
```suggestion
// then
        Awaitility.await()
                .atLeast(1, TimeUnit.SECONDS)
                .untilAsserted(() -> {
                    Campaign modifiedCampaign = campaignRepository.findById(findCampaign.getId())
                            .orElseThrow();
                    assertThat(modifiedCampaign.getState()).isEqualTo(CampaignState.IN_PROGRESS);
                });
```
페어 프로그래밍 당시 작성했던 await 코드가 어떻게 작성되었는지는 모르겠으나 다음과 같이, 작성했을 때 테스트가 통과하였습니다.